fixtures/workers-with-assets-and-service-bindings/workerWS/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import { WorkerEntrypoint } from "cloudflare:workers";
+
+export default class extends WorkerEntrypoint {
+	fetch(req: Request) {
+		if (!req.headers.get("Upgrade")?.includes("websocket")) {
+			return new Response("Hello from worker-ws fetch()");
+		}
+
+		const [server, client] = Object.values(new WebSocketPair());
```

Comment:
```ts
  const client = webSocketPair[0],
    server = webSocketPair[1];
```

That's what you are doing differently ;)

Reply:
the docs example will work towards what I have in my code

```
The WebSocketPair constructor returns an Object, with the 0 and 1 keys each
holding a WebSocket instance as its value. It is common to grab the two WebSockets 
from this pair using Object.values ↗ and ES6 destructuring ↗, as seen in the below 
example.
```